src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -361,25 +361,25 @@
     <value>The string literal parameter of an attribute does not parse correctly for a URL, a GUID, or a version.</value>
   </data>
   <data name="AttributeStringLiteralsShouldParseCorrectlyMessageDefault" xml:space="preserve">
-    <value>In the constructor of '{0}', change the value of argument '{1}', which is currently "{2}", to something that can be correctly parsed as '{3}'.</value>
+    <value>In the constructor of '{0}', change the value of argument '{1}', which is currently "{2}", to something that can be correctly parsed as '{3}'</value>
   </data>
   <data name="AttributeStringLiteralsShouldParseCorrectlyMessageEmpty" xml:space="preserve">
-    <value>In the constructor of '{0}', change the value of argument '{1}', which is currently an empty string (""), to something that can be correctly parsed as '{2}'.</value>
+    <value>In the constructor of '{0}', change the value of argument '{1}', which is currently an empty string (""), to something that can be correctly parsed as '{2}'</value>
   </data>
   <data name="AvoidZeroLengthArrayAllocationsTitle" xml:space="preserve">
-    <value>Avoid zero-length array allocations.</value>
+    <value>Avoid zero-length array allocations</value>
   </data>
   <data name="AvoidZeroLengthArrayAllocationsMessage" xml:space="preserve">
     <value>Avoid unnecessary zero-length array allocations.  Use {0} instead.</value>
   </data>
   <data name="DoNotUseEnumerableMethodsOnIndexableCollectionsInsteadUseTheCollectionDirectlyTitle" xml:space="preserve">
-    <value>Do not use Enumerable methods on indexable collections. Instead use the collection directly</value>
+    <value>Do not use Enumerable methods on indexable collections</value>
```

Comment:
I have checked and this is the expected behavior of your code-fix.

Reply:
Yes, this looks to be the correct change.